app/lib/three_scale/redis_config.rb

```diff
@@ -4,16 +4,14 @@ module ThreeScale
   class RedisConfig
     def initialize(redis_config = {})
       raw_config = (redis_config || {}).deep_symbolize_keys
-      sentinels = raw_config.delete(:sentinels).presence
-      raw_config.delete_if { |key, value| value.blank? }
+      raw_config.delete_if { |_key, value| value.blank? }
       uri = URI.parse(raw_config[:url].to_s)
       raw_config[:db] ||= uri.path[1..]
-      raw_config[:name] ||= uri.host if sentinels
       raw_config[:ssl] ||= true if uri.scheme == 'rediss'
+      parse_sentinels(raw_config)
```

Comment:
Just a suggestion - maybe make the name of the method more clear to reflect its intentions. Currently it does not only parse and return something, but actually changes the passed configuration. So, maybe either add a `!` to indicate it modifies the argument, or change the name, or both :)

Reply:
Done at https://github.com/3scale/porta/pull/3855/commits/e054b422a010e14502f52997a29eea7132770b47. Hope the new name is better!